installing/installing_aws/installing-aws-outposts-remote-workers.adoc

```diff
@@ -0,0 +1,71 @@
+:_content-type: ASSEMBLY
+[id="installing-aws-default"]
+= Installing a cluster on AWS with remote workers on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts-remote-workers
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Amazon Web Services (AWS) with remote workers running on AWS Outposts.
+This can be achieved by customizing the default AWS installation and performing some manual steps.
```

Comment:
Do we want to say that we are referring do ipi installation?

Reply:
This was the format used in other AWS installation instuctions, the fact that "referring to IPI installation" wasn't mentioned there